src/main/java/org/example/competitionService/CompetitionController.java

```diff
@@ -0,0 +1,23 @@
+package org.example.competitionService;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.example.Competition;
+
+public class CompetitionController {
+    private final ObjectMapper objectMapper;
+
+    public CompetitionController() {
+        objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+    }
+
+    public Competition createCompetitionFromJson(String jsonRequest) {
+        try {
+            return objectMapper.readValue(jsonRequest, Competition.class);
```

Comment:
it is not creation. Actually it is converting json into object. I guess your service should create converted from json Competition